### PR TITLE
Js fixbug

### PR DESCRIPTION
**Pull request by jrsewell400**
Made the following changes:
- Split the navigation tests into two separate files(standard nav and nav restrictions), and created a navigation test folder where they both reside.
- Reworked the item delete and edit tests, to account for merchant employees and admins only being allowed to modify items.
- Added view logic to only allow only merchant employees and admins to edit and delete items.
- Add test for and text to welcome page.
- Fixed placement of Logged In within nav bar
- Added line breaks, etc. for aesthetics.

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>WELCOME TO MONSTER SHOP</h1>
```
Comment (elguapogordo) [inline app/views/welcome/index.html.erb:1]: dynamic!

**Review by elguapogordo — APPROVED** [review]

File: app/views/items/show.html.erb [hunk 2]
```diff
@@ -17,44 +17,46 @@
       <% else %>
         <p style= "color:red">Inactive</p>
       <% end %>
-      <p><%= link_to "Edit Item", "/items/#{@item.id}/edit" %></p>
-      <p><%= link_to "Delete Item", "/items/#{@item.id}", method: :delete if @item.no_orders?%></p>
       <%= link_to "Add Review", "/items/#{@item.id}/reviews/new" %>
+      <% if current_merchant? || current_admin? %>
```
Comment (zachholcomb) [inline app/views/items/show.html.erb:21]: nice i used that same logic to guard against regular users as well

File: app/views/items/show.html.erb [hunk 3]
```diff
@@ -17,44 +17,46 @@
...
+        <p><%= link_to "Delete Item", "/items/#{@item.id}", method: :delete if @item.no_orders?%></p>
+        <p><%= link_to "Edit Item", "/items/#{@item.id}/edit" %></p>
+      <% end %>
       <% if !current_admin? %>
```
Comment (steve-meyers) [inline app/views/items/show.html.erb:25]: This is a good catch and it makes sense that only an admin and merchant can do those things.

File: app/views/items/show.html.erb [hunk 4]
```diff
@@ -17,44 +17,46 @@
...
+      <% @item.reviews.each do |review| %>
+        <section id= 'review-<%=review.id%>'>
+          <%= review.title %></br>
+          <%= review.content %></br>
+          Rating: <%= review.rating %>/5</br>
+          <%= link_to "Edit", "/reviews/#{review.id}/edit" %></br>
+          <%= link_to "Delete", "/reviews/#{review.id}", method: :delete %></br></br>
+        </section>
+        <% end %>
     <% end %>
```
Comment (steve-meyers) [inline app/views/items/show.html.erb:60]: Have we tested the functionality of reviews at all?

File: spec/features/navigation/nav_restriction_spec.rb [hunk 5]
```diff
@@ -1,111 +1,6 @@
...
-      visit '/login'
-      fill_in :Email, with: user.email
-      fill_in :Password, with: user.password
-      click_button "Login"
-      visit '/admin/dashboard'
-      expect(page).to have_content("The page you were looking for doesn't exist (404)")
-    end
-  end
-
+RSpec.describe "Navigation Restrictions", type: :feature do
```
Comment (zachholcomb) [inline spec/features/navigation/nav_restriction_spec.rb:3]: nice looks good here, glad we moved these into two tests it makes a little more sense to me

File: spec/features/items/merchant/item_destroy_spec.rb [hunk 6]
```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
```
Comment (zachholcomb) [inline spec/features/items/merchant/item_destroy_spec.rb:2]: like splitting tests into smaller chunks here

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -28,8 +28,8 @@
         <% elsif current_user.admin? %>
           <%= link_to "Admin Dashboard", "/admin/dashboard" %>
           <%= link_to "See all Users", "/admin/users" %>
-        <span>Logged in as <%= current_user.name %></span>
         <% end %>
+        <span>Logged in as <%= current_user.name %></span>
       <% else %>
```
Comment (steve-meyers) [inline app/views/layouts/application.html.erb:33]: This looks much better now.

File: app/views/welcome/index.html.erb (hunk 1 above)
Comment (steve-meyers) [reply app/views/welcome/index.html.erb:1]: @elguapogordo Technically this is static.

**Review by steve-meyers — APPROVED** [review]
Everything looks good!

**Pull request merged**